packages/block-library/src/pattern/index.php

```diff
@@ -22,14 +22,22 @@ function register_block_core_pattern() {
 /**
  * Renders the `core/pattern` block on the server.
  *
- * @param array $attributes Block attributes.
+ * @param array  $attributes Block attributes.
+ * @param string $content    The block rendered content.
  *
  * @return string Returns the output of the pattern.
  */
-function render_block_core_pattern( $attributes ) {
+function render_block_core_pattern( $attributes, $content ) {
 	if ( empty( $attributes['slug'] ) ) {
 		return '';
 	}
+	$slug_classname = str_replace( '/', '-', $attributes['slug'] );
+	$classnames     = isset( $attributes['className'] ) ? $attributes['className'] . ' ' . $slug_classname : $slug_classname;
+	$wrapper        = '<div class="' . esc_html( $classnames ) . '">%s</div>';
```

Comment:
```suggestion
	$wrapper        = '<div class="' . esc_attr( $classnames ) . '">%s</div>';
```

This should be using `esc_attr` to properly escape the HTML attribute.